test/database_service/test_db_experiment_data.py

```diff
@@ -0,0 +1,724 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+# pylint: disable=missing-docstring
+
+"""Test ExperimentData."""
+
+import os
+from unittest import mock, skipIf
+import copy
+from random import randrange
+import time
+import threading
+import json
+import re
+import uuid
+
+import numpy as np
+
+from qiskit.test import QiskitTestCase
+from qiskit.test.mock import FakeMelbourne
+from qiskit.result import Result
+from qiskit.providers import JobV1 as Job
+from qiskit.providers import JobStatus
+from qiskit.tools.visualization import HAS_MATPLOTLIB
+from qiskit_experiments.database_service import DbExperimentDataV1 as DbExperimentData
+from qiskit_experiments.database_service import DatabaseServiceV1
+from qiskit_experiments.database_service.exceptions import (
+    DbExperimentDataError,
+    DbExperimentEntryNotFound,
+    DbExperimentEntryExists,
+)
+
+
+class TestDbExperimentData(QiskitTestCase):
+    """Test the DbExperimentData class."""
+
+    def setUp(self):
+        super().setUp()
+        self.backend = FakeMelbourne()
+
+    def test_stored_data_attributes(self):
+        """Test stored data attributes."""
+        attrs = {
+            "job_ids": ["job1"],
+            "share_level": "global",
+            "figure_names": ["figure1"],
+            "notes": "some notes",
+        }
+        exp_data = DbExperimentData(
+            backend=self.backend,
+            experiment_type="qiskit_test",
```

Comment:
OK this is probably sufficient to let us know if these were ever part of an integration test that we could cleanup orphans on the real service.